lib/functional_mixin.js

```diff
@@ -328,6 +328,25 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  // Waits for target element to be not empty
+  testCase.waitForElementToContain = function busterFunctionalMixin_waitForElementToContain(selector, callback)
+  {
+    // check if the target element is present & and it's not empty
```

Comment:
Can you add there following string? `target = typeof selector == 'string' ? this.$(selector) : selector` so if jQuery object passed to the function, it will work too? Thank you.